src/classes/RD2_RecurringDonationsOpp_TDTM.cls

```diff
@@ -75,11 +75,16 @@ public class RD2_RecurringDonationsOpp_TDTM extends TDTM_Runnable {
         }
 
         Set<Id> rdIds = new Set<Id>();
```

Comment:
Since this trigger will get every single new Opportunity record, what would you think about first checking the trigger.new collection to see if there are any records with a value in the `npe03__Recurring_Donation__c` field before calling any of the other methods? I'm mostly thinking about reducing any additional processing in an already crowded trigger framework for Opps, where generally only a small fraction of the opps in an org will actually be linked to an RD record.

Reply:
Thank you